lib/config/package.js

```diff
@@ -213,6 +213,12 @@ PackageJSON.prototype.write = function() {
       if (this.originalPjson[p] === pjson[p])
         delete pjson[p];
     }
+
+    // keep original overrides
+    try {
+      pjson.overrides = this.originalPjson.jspm.overrides;
```

Comment:
Follow the same pattern as in https://github.com/jspm/jspm-cli/blob/master/lib/config/package.js#L95 so that this becomes `config.pjson.overrides` when accessed from install.js.